docs/azure/get-started-with-azure/guardrails.md

```diff
@@ -0,0 +1,220 @@
+# Azure landing zone guardrails: What you need to know
+
+Last updated: **{{ git_revision_date_localized }}**
+
+This document explains the "guardrails" (security and compliance rules) set up in our Azure Landing Zone. These guardrails are implemented using Azure Policy, and they are in place to ensure our cloud environment remains secure, compliant (including with standards like **CCCS Medium [Canada Federal PBMM]**), and cost-effective. They help us maintain consistency and prevent configurations that could lead to problems.
+
+Think of these guardrails as automatic checks and balances. They don't require you to do anything extra most of the time, but they might prevent you from doing something that goes against our established best practices, or they might automatically configure resources to meet compliance standards.
+
+!!! info
+    This document focuses on the things that you might be *prevented* from doing (Deny policies), the things you *must* do as a result, or things that will be automatically configured (DeployIfNotExists and Modify policies). Audit policies are also in place, which will flag non-compliant resources, but won't block deployments.
+
+## Key areas covered
+
+1.  **Resource Deployment Restrictions:** Where and what you can deploy.
+2.  **Networking Restrictions:** How your networks are configured and protected.
+3.  **Security Requirements:** Mandatory security settings.
+4.  **Data Protection:** How your data is protected at rest and in transit.
+5.  **Cost Optimization:** Preventing unnecessary costs.
+6.  **Identity & Access Management:** Controlling who can access resources.
+7.  **Specific Service Guardrails:** Restrictions related to specific Azure services.
+8.  **Monitoring:** Diagnostic data collection and automated security checks.
+9.  **Tagging:** Automatic application of tags.
+
+## Resource deployment restrictions
+
+*   **Allowed Locations (Regions):** You can only deploy resources in Canada Central and Canada East. If you try to deploy to a region not on the approved list, the deployment will be **blocked**. Note that networking is only provided in Canada Central.
+*   **Resource Group Locations:** The location (region) of a resource group *must* match the location of the resources within it.
```

Comment:
This is actually backwards. You can't create a resource without a Resource Group. So the RG needs to exist first.